src/docs.rs

```diff
@@ -17,7 +17,7 @@ use askama::Template;
 use itertools::Itertools;
 use std::path::Path;
 
-const MAX_COLUMNS: isize = 65;
+pub const MAX_COLUMNS: isize = 65;
```

Comment:
This seems like it might be a poor change to make; I'd be happy to take an alternate route

Reply:
I think the use of a constant is good, but for the other Erlang we're using 80.

https://github.com/gleam-lang/gleam/blob/6d2ce262451a268739c2e44c1f42a6df825c57d5/src/erl.rs#L401

Let's extract that into a constant and share it between these two places? Keeping the docs one private.